examples/hello_world_component/Makefile

```diff
@@ -6,23 +6,21 @@
 .PHONY: all
 
 ifeq ($(BUILD_TYPE),Debug)
-  hpx_application = hpx_application_debug
-  hpx_component = hpx_component_debug
   debug_suffix = d
 else
-  hpx_application = hpx_application
-  hpx_component = hpx_component
   debug_suffix =
 endif
+hpx_application = hpx_application
+hpx_component = hpx_component
 
 APP_CFLAGS = $(CXX_FLAGS)
 APP_CFLAGS += $(shell PKG_CONFIG_PATH=$(HPX_DIR) $(PKG_CONFIG) --cflags --libs $(hpx_application))
-APP_CFLAGS += -lhpx_iostreams$(debug_suffix)
+#APP_CFLAGS += -lhpx_iostreams$(debug_suffix)
 APP_CFLAGS += -DHPX_APPLICATION_NAME=test
 
 COMP_CFLAGS = $(CXX_FLAGS)
 COMP_CFLAGS += $(shell PKG_CONFIG_PATH=$(HPX_DIR) $(PKG_CONFIG) --cflags --libs $(hpx_component))
-COMP_CFLAGS += -lhpx_iostreams$(debug_suffix)
+#COMP_CFLAGS += -lhpx_iostreams$(debug_suffix)
```

Comment:
Uncomment?

Reply:
For now I was just adding all "to be used" components to the linking step, but I'll change that in a `-L` with the components directories and leave the `-l` to be specified by the user